### ISSUE TITLE
Naming (and semantics) of relatedLink and significantLink

### ISSUE THREAD

**Issue by dmejia** [schemaorg/schemaorg#875]
http://schema.org/WebPage supports relatedLink and significantLink which are both of type URL. I don't see, however, the concept of links used anywhere else, and the naming is usually *Url. Moreover, to be more precise semantically shouldn't the relationship be between web pages? Basically a WebPage can be related to other pages or link to other pages, so instead we could have something like

relatedPage and significantSubPage or significantLinkedPage (of type URL or WebPage).

(Practically you could then provide additional information about the linked pages beyond the URL).

**Comment by VladimirAlexiev** [comment]
I agree that the range of both props should be expanded to include `WebPage`.

I also have a similar question about the semantics of the two:
- https://schema.org/significantLink is more clear to me: it's a link to another page, not part of the standard header/footer/navbar/menu links
- but https://schema.org/relatedLink:
  - description is imprecise: "A **link related** to this web page, for example to other related web pages."
     IMHO links cannot be related: the content denoted by the link can be related. So I suggest to reword to:
     "A link on this web page that points to related content, be that WebPage (with some semantic datA), or just URL (with no semantic data"
  - is it weaker than `significantLink`?

**Comment by jvandriel** [comment]
I recently tried to look into the history of `significantLink` and `relatedLink` but unfortunately there are no public records containing any information regarding the reasoning behind their existence. Not even a resource can be found about `significantLink`'s update in schema.org version 0.91 (2012-04-21).

Meaning it is impossible for say why they were added to the vocabulary at the start and for which intended purposes. As far as I am aware though no large data consumer has ever documented making use of them. 

Personally I treat them as remnants of the past, pretty much the same way dictionaries contain words we also no longer use (yet might be used again in the future).

**Comment by VladimirAlexiev** [comment]
But @jvandriel surely at least the internal links between WebPages of a site is important enough info to be captured?

**Comment by jvandriel** [comment]
Generally speaking publishing markup is about making statements about the entities that are contained within web pages, and not so much about providing metadata about web pages themselves (with a few exceptions like `breadcumb` or `ProfilePage`, etc).

Having said that, there's nobody stopping us from publishing the things we want to if we have our own purposes for it, like you described for Ontotext.

I have added plenty of 'stuff' to markup in the past for my own analytical reasons, which is a totally valid use case but it doesn't imply it's therefor also useful for other data consumers.

**Comment by VladimirAlexiev** [comment]
Schema lacks a general relation like `dct:related` or `skos:relatedMatch`. 

`significantLink` and `relatedLink` are IMHO the best candidates for such relation 
(the former being stronger than the latter, similar to `about` vs `mentions`).

I was recently told by a SEO guy that each of our pages should be represented by 2 items: Article and WebPage, with `mainEntityOfPage` link between them.
So @jvandriel in regards to your comment I stand corrected: I want to use these props to link related pieces of content (Articles), not WebPages.

Therefore the domain of the two props should be raised to `CreativeWork`, and the range should include `URL, CreativeWork`

**Comment by smrgeoinfo** [comment]
this is a glaring gap in schema.org-- no clear way to express a typed relationship between any schema:Thing.  This is the 'cop out' we've been using for CDIF:
```
...
   "@id": "ex:cdifOptional_23578",
    "@type": ["schema:Dataset"],
...
    "schema:relatedLink": [
        {
            "@type": "schema:LinkRole",
            "schema:linkRelationship": "related visualization",
            "target": {
                "@type": "schema:EntryPoint",
                "schema:encodingFormat": "image/jpg",
                "schema:name": "Image of seafloor geology map, bay of Biscay",
                "schema:url": "https://example.org/geology/baybiscay"
            }
        },
        {
            "@type": "schema:LinkRole",
            "schema:linkRelationship": {
                "@type": "schema:DefinedTerm",
                "schema:name": "related dataset",
                "schema:inDefinedTermSet": "https://www.iana.org/assignments/link-relations/",
                "schema:termCode": "related"
            },
            "target": {
                "@type": "schema:EntryPoint",
                "schema:name": "Bay of Biscay current velocity dataset",
                "schema:url": "https://example.org/currents/baybiscay"
            }
        }
    ],
```
and just ignoring the validatory warnings and errors....
